taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+from django import forms
+from .models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = get_user_model()
+        fields = ("username", "license_number", "first_name", "last_name",)
+
+
+class DriverUpdateForm(forms.ModelForm):
+    class Meta:
+        model = get_user_model()
+        fields = ("username", "license_number", "first_name", "last_name",)

```

Comment:
Similar to the `DriverCreationForm`, ensure that the `license_number` field exists in the user model returned by `get_user_model()`.